pkg/volume/configmap/configmap.go

```diff
@@ -266,9 +266,9 @@ func MakePayload(mappings []v1.KeyToPath, configMap *v1.ConfigMap, defaultMode *
 				if optional {
 					continue
 				}
-				err_msg := "references non-existent config key"
-				glog.Errorf(err_msg)
```

Comment:
it doesn't seem to make very much sense to log this. removed. PTAL @smarterclayton